monitoring-service/src/main/java/org/terracotta/management/service/monitoring/MonitoringServiceProvider.java

```diff
@@ -42,7 +46,8 @@ public boolean initialize(ServiceProviderConfiguration configuration) {
     MonitoringServiceConfiguration config = configuration instanceof MonitoringServiceConfiguration ?
         (MonitoringServiceConfiguration) configuration :
         new MonitoringServiceConfiguration();
-    this.monitoringService = new MonitoringService(config);
+    SequenceGenerator generator = new BoundaryFlakeSequenceGenerator(TimeSource.BEST, NodeIdSource.BEST);
```

Comment:
should use default constructor ? just to emphasize that people should use that anyway